daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -1277,6 +1277,15 @@ object SBuiltin {
     }
   }
 
+  /** $ty_con
+    *     :: Text
+    */
+  final case class SBTyCon(tyCon: TypeConName) extends SBuiltin(0) {
```

Comment:
`SBTyCon` sounds slightly confusing. Maybe `SBTyConRep` would be better? Depending on whether we limit this primitve to template types or not we could also call this `SBTemplateTyRep`.

Reply:
I've renamed it to `to_text_template_id` (`ToTextTemplateId`).

Reply:
:+1: